core/service/src/chain_spec.rs

```diff
@@ -29,15 +30,15 @@ use tel::TelemetryEndpoints;
 
 enum GenesisSource<G> {
 	File(PathBuf),
-	Embedded(&'static [u8]),
+	Embedded(Cow<'static, [u8]>),
```

Comment:
```suggestion
	Binary(Cow<'static, [u8]>),
```

Or something different, but embedded sounds not that good.